ghost/content.md

```diff
@@ -17,3 +17,13 @@ If you'd like to be able to access the instance from the host without the contai
 	docker run --name some-ghost -p 8080:2368 -d ghost
 
 Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+
+You can also point the image to your existing content on your host:
+
+	docker run --name some-ghost -v /path/to/ghost/blog:/var/lib/ghost ghost
+
+Alternatively you can use a data container that points to ```/var/lib/ghost```:
+
+	docker create --name some-ghost-data -v /var/lib/ghost ghost /bin/true
```

Comment:
If the image being used to create the data container is `ghost`, it already has a `/var/lib/ghost` volume in the Dockerfile. Not sure if this will confuse users or not.

I've generally seen people create "data containers" with images like `tianon/true`, in which case you *would* need to declare the volume yourself as you've done in your `docker create` command.